docs/src/pages/demos/cards/cards.md

```diff
@@ -33,6 +33,8 @@ By default, we use the combination of a `<div>` element and a *background image*
 
 {{"demo": "pages/demos/cards/ImgMediaCard.js"}}
 
+> ⚠️ The previous demo replies on `object-fit` for centering the image. It's not supported by IE 11.
```

Comment:
```suggestion
> ⚠️ The previous demo relies on `object-fit` for centering the image. It's not supported by IE 11.
```

Reply:
Would it not be more correct saying CardMedia relies on `object-fit`?

Reply:
👍